client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -239,17 +262,23 @@ const CheckoutThankYou = React.createClass( {
 export default connect(
 	( state, props ) => {
 		return {
-			receipt: getReceiptById( state, props.receiptId )
+			receipt: getReceiptById( state, props.receiptId ),
+			sitePlans: getPlansBySite( state, props.selectedSite )
 		};
 	},
 	( dispatch ) => {
 		return {
-			activatedTheme: ( meta, selectedSite ) => {
-				dispatch( activated( meta, selectedSite, 'calypstore', true ) );
+			activatedTheme: ( meta, site ) => {
+				dispatch( activated( meta, site, 'calypstore', true ) );
 			},
 			fetchReceipt: ( receiptId ) => {
 				dispatch( fetchReceipt( receiptId ) );
 			},
+			fetchSitePlans( sitePlans, site ) {
```

Comment:
What do you think about moving this action to `FreeTrialNudge`? This component isn't concerned with the site plans data as far as I can tell, and we could avoid adding `sitePlans` to the props for this component if we moved this action.